ethportal-peertest/src/scenarios/gossip.rs

```diff
@@ -90,37 +91,99 @@ pub async fn test_gossip_dropped_with_offer(peertest: &Peertest, target: &Client
         .unwrap();
     let fresh_enr = fresh_target.node_info().await.unwrap().enr;
 
-    // Store accumulator_1 locally in client that is not connected to the network
-    let (acc_key_1, acc_value_1) = fixture_epoch_acc_1();
+    // Store block_1 locally in client that is not connected to the network
```

Comment:
Sorry, but I realize now that the large content values I gave were not ideal, and it shows in the complexity of this test, making it difficult to follow what's going on...
I updated #1438 with new assets that should simplify this test... 
You should be able to write a test that's accurately testing the gossip-dropped feature using only the..
- receipts from block `15040641` (320kb)
- body from block  `15040708` (763kb)
- (and the 2 headers for validation)
As it stands, there's just too much going on and it's tough to follow the exact code paths that are being tested